packages/screens/AdministrationDashboard/component/CarouselHero.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { Image, View, ViewStyle, useWindowDimensions } from "react-native";
+import Carousel from "react-native-reanimated-carousel";
+
+import guardianPng from "../../../../assets/default-images/guardian_1.png";
+import {
+  neutral00,
+  neutral44,
+  secondaryColor,
+} from "../../../utils/style/colors";
+import { layout } from "../../../utils/style/layout";
+
+const MD_BREAKPOINT = 820;
+
+export const CarouselHero = () => {
+  const { width } = useWindowDimensions();
+  const heroData = [...new Array(6).keys()];
+  const isBreakPoint = width >= MD_BREAKPOINT;
+  const [currentPage, setCurrentPage] = useState<number>(0);
+
+  return (
+    <View
+      style={[
+        {
+          marginRight: layout.spacing_x4,
+          position: "relative",
+          alignItems: "center",
+          justifyContent: "center",
+        },
+        isBreakPoint ? { flex: 1 } : { marginTop: 16 },
+      ]}
+    >
+      <Carousel
```

Comment:
To be interactable i think you can add this props `panGestureHandlerProps={{ enableTrackpadTwoFingerGesture: true }}`
And i think you can add `autoPlayInterval={7000}` to automatically pass to an other hero, but for that i think you have to handle the order of the carousel and for that you have `onProgressChange` props that can help you 👍